File: bindings/java/src/org/raml/amf/parsers/RAMLParser.java
package org.raml.amf.parsers;

/**
 * Created by antoniogarrote on 04/05/2017.
 */

/**
 * Wrapper class for the AMF RAML parser, processes RAML specification documents and generate the DocumentModel out of them
 */
public class RAMLParser extends BaseParser {

    @Override
    protected String parserConstructor() {
        return "->RAMLParser";
    }
}

File: bindings/js/src/parsers/RAMLParser.ts
/**
 * Created by antoniogarrote on 05/05/2017.
 */

import {BaseParser} from "./BaseParser";
import {Clojure} from "../Clojure";

/**
 * Wrapper class for the AMF RAML parser, processes RAML specification documents and generate the DocumentModel out of them
 */
export class RAMLParser extends BaseParser {
    protected parser(): any {
        return new Clojure.amf.__GT_RAMLParser();
    }
}